frontend/src/components/CreateCollectionModal/components/Content/components/LinkInput/index.tsx

```diff
@@ -61,17 +95,50 @@ const LinkInput: FC<Props> = ({
           onClick={() => handleDelete(id)}
         />
       </IconWrapper>
-    </Wrapper>
+    </LinkWrapper>
   );
 
+  function handleNameChange({
```

Comment:
Every field change needs to fire the parent's handle link change so the links array is updated with the updated link